### PR TITLE
added a header for usage with process.env

### PR DESCRIPTION
**Pull request by mwarger**
added a header for usage with process.env to separate section from 'current lifecycle event' and make it easier to find

### CODE REVIEW

**Review by zkat — COMMENTED** [review]
I'm not really sure this header is needed -- reading this section a few times, it seems clear that the section under your header is directly relevant to its original location, and I think I'd much prefer keeping the docs as they are, in this case?

**Comment by terinjokes** [comment]
The Twitter thread that spawned this was in relation to the "package.json vars" section, and the lack of a usage example.

If we're going to be changing the docs, I agree with @zkat that creating a new header here doesn't make sense.

Perhaps add an example to the other section showing off `npm_package_name` or `npm_package_version`?

**Comment by mwarger** [comment]
I added the header because that paragraph (the one I put the header above) seemed to apply to all the previous paragraphs.  The flattening of the object structure with the subsequent description of how to use it.

That's a good point, though - I will try your suggestion.  Thank you for the feedback.

File: doc/misc/npm-scripts.md [hunk 1]
```diff
@@ -143,7 +143,11 @@ The package.json fields are tacked onto the `npm_package_` prefix. So,
 for instance, if you had `{"name":"foo", "version":"1.2.5"}` in your
 package.json file, then your package scripts would have the
 `npm_package_name` environment variable set to "foo", and the
-`npm_package_version` set to "1.2.5"
+`npm_package_version` set to "1.2.5".
+
+#### Usage
```
Comment (neverett) [inline doc/misc/npm-scripts.md]: For consistency with the other sections, I would recommend moving the example up to follow the previous paragraph, and changing the language a bit. Here's how that might look:

>  The package.json fields are tacked onto the `npm_package_` prefix. So, for instance, if you had `{"name":"foo", "version":"1.2.5"}` in your package.json file, then your package scripts would have the `npm_package_name` environment variable set to "foo", and the `npm_package_version` set to "1.2.5". You can access these variables in your code with `process.env.npm_package_name` and `process.env.npm_package_version`, and so on for other fields.

**Review by neverett — COMMENTED** [review]
One comment on the placement and wording of the example -- let me know what you think.

**Comment by mwarger** [comment]
Good call - I made that change.  Thanks all for the feedback.

**Comment by mwarger** [comment]
This was just a text change - can the CI be kicked off again?  I don't think my change caused this...

**Review by zkat — APPROVED** [review]
👍 changes look good to me. Thank you!

And don't worry about CI. It's not your fault ^_^;;

**Pull request merged**